db/migrate/20191210171030_add_offer_course_option_id_to_application_choice.rb

```diff
@@ -0,0 +1,5 @@
+class AddOfferCourseOptionIdToApplicationChoice < ActiveRecord::Migration[6.0]
+  def change
+    add_column :application_choices, :offer_course_option_id, :integer
```

Comment:
Perhaps `offered_course_option_id`? Perhaps this can use `add_reference` to generate foreign key constraints.

Reply:
Migration is probably going away if we store it in JSON 👍

+1 for the FK suggestion if not.